DSharpPlus/Entities/Channel/Message/Poll/DiscordPollBuilder.cs

```diff
@@ -27,9 +27,9 @@ public class DiscordPollBuilder
     private readonly List<DiscordPollMedia> _options = new();
     
     /// <summary>
-    /// Gets or sets the expiry date for this poll.
+    /// Gets or sets the duration for this poll in hours.
     /// </summary>
-    public DateTimeOffset Expiry { get; set; } = DateTimeOffset.UtcNow.AddHours(1);
+    public int Duration { get; set; } = 1;
```

Comment:
is there any reason we're using an integer and not a TimeSpan?

Reply:
Discord takes an integer called 'Duration' that represents the amount of hours the poll is up when creating the poll.
I could turn this into a TimeSpan if that's preferred.

Reply:
Is this still something you'd want to change? It doesn't make sense to make it a TimeSpan to me if discord will only accept an int that represents hours.